### ISSUE TITLE
Player stops opening after a while

### ISSUE THREAD

**Issue by chimlm** [Tzahi12345/YoutubeDL-Material#64]
The link on the thumb tiles representing downloaded videos supposed to open the player page for it. That works for a few times. At some point clicking on them the browser is doing nothing. When this happen it happens to all videos no matter played before or not. Used latest version of Firefox and Edge on Windows 10. I try the Firefox inspector and it seems the browser is sending a getMP4 request, gets a 304, then sit still.

**Comment by Tzahi12345** [comment]
I think I may be replicating this. Is multi-download mode on when it fails? Whenever I enable it, I'm unable to navigate to videos.

**Comment by chimlm** [comment]
yes, it seems selecting the multi-download checkbox with also disable the video player. Hope this is a easy fix.

**Comment by Tzahi12345** [comment]
It was quite easy to fix actually, you can view the relevant [commit here](https://github.com/Tzahi12345/YoutubeDL-Material/commit/d2af233a1fa72a205565c3b19664ab4de69b330e).

You can test out the fix in the latest `nightly` Docker builds or wait until the next update, Anyways, thanks for bringing up this bug! I'll go ahead and close this issue.
